### DISCUSSION TITLE
Regarding Incremental Classifier - Output Unit

### DISCUSSION THREAD

**Discussion by AdityaJainTD** [ContinualAI/avalanche#1109] [Q&A] [answered]
Hi everyone,

I was experimenting with Incremental Classifiers. I came across the documentation which says these type of classifiers add 2 new units for the unseen class. Can anyone explain why we need to add 2 units? Is there any significant attached to it? Wont it work just by adding a single unit?

-Thanks

**Comment by AntonioCarta** [comment, marked as answer]
where did you read this? the IncrementalClassifier adds units for new classes (a single unit for each class).

**Reply by AdityaJainTD** [reply]
Here is the link (https://avalanche.continualai.org/v/avalanche/from-zero-to-hero-tutorial/02_models), it says 2 units under the section 'Dynamic Model Expansion'

**Reply by AdityaJainTD** [reply]
@AntonioCarta Can you please provide me any update on this?

**Reply by AntonioCarta** [reply]
Hi @AdityaJainTD. The docs uses Split MNIST with 5 experiences. Therefore, you have two new classes for each experience, which is why the model adds two units at each step.

**Reply by AdityaJainTD** [reply]
@AntonioCarta Thanks for the clarification.
